lib/kitchen/directions/bake_chapter_summary.rb

```diff
@@ -15,7 +15,6 @@ def self.v1(chapter:, metadata_source:, klass: 'summary', uuid_prefix: '.')
       end
 
       class V1
-        renderable
         def bake(chapter:, metadata_source:, uuid_prefix: '.', klass: 'summary')
           summaries = Clipboard.new
 
```

Comment:
I baked pl-psychology on this branch and I noticed additional attribute in H3 headers from Summary:

![image](https://user-images.githubusercontent.com/40228854/125062384-9e516d80-e0ae-11eb-990d-2bb64543357f.png)

Probably it's related with previous changes, not specifically with this PR :thinking:

Reply:
Good catch... 🤔 You're right, [this is where that change was introduced.](https://github.com/openstax/kitchen/pull/280/files#diff-ee054ac0aca7a439df006e3f489616ea44c8785f0e188b0577b8ac2f54642b44R28-R30) 

I manually added the `itemprop` attribute because it showed up in the test data; `EocTitleLinkSnippet` doesn't add it. Now I'm wondering if we should change the implementation of `EocTitleLinkSnippet`? It looks like, for the `BakeChapterSummary` direction, this title was being added with `page.title.copy`, instead of being manually constructed as it is in `EocTitleLinkSnippet`, which is probably letting it have the full set of attributes.

Should we tackle this in a separate PR?

Reply:
Yes, I think it's a good idea to take care of this in a separate PR :+1: Probably we should investigate why section title link was added in a different way in Summary. If it was intentional or someone just came up with different idea.